cg/constants/pacbio.py

```diff
@@ -0,0 +1,19 @@
+"""Constants related to PacBio sequencing."""
+
+
+class PacBioDirsAndFiles:
+    ADAPTER_REPORT: str = "adapter.report.json"
+    CCS_REPORT: str = "ccs.report.json"
+    CONTROL_REPORT: str = "control.report.json"
+    LOADING_REPORT: str = "loading.report.json"
+    RAW_DATA_REPORT: str = "raw_data.report.json"
+
+
+class CCSAttributeIDs:
```

Comment:
And this @Vince-janv 
```suggestion
class HiFiAttributeIDs:
```